README.md

```diff
@@ -47,24 +47,24 @@ To optimize our service, we divided our consultants into three regions. Consider
 - **ESPECIAL**
 
 ```
-minlon: -2.196998
+minlon: -15.411580
 minlat -46.361899
-maxlon: -15.411580
+maxlon: -2.196998
 maxlat: -34.276938
 ```
 ```
-minlon: -19.766959
+minlon: -23.966413
```

Comment:
To insert a empty state on special category:
```suggestion
minlon: -20.000000
```

Reply:
Done: https://github.com/juntossomosmais/frontend-challenge/pull/4/commits/beb9cc3e3e6d6e50ddcdad60b4967babc2b54191